xbmc/interfaces/legacy/Addon.h

```diff
@@ -56,78 +75,145 @@ namespace XBMCAddon
       String getAddonVersion();
 
     public:
-      /**
-       * Addon class.
-       * 
-       * Addon([id]) -- Creates a new Addon class.
-       * 
-       * id          : [opt] string - id of the addon as specified in addon.xml\n
-       * 
-       * *Note, specifying the addon id is not needed.\n
-       *  Important however is that the addon folder has the same name as the addon id provided in addon.xml.\n
-       *  You can optionally specify the addon id from another installed addon to retrieve settings from it.
-       * 
-       * example:
-       *  - self.Addon = xbmcaddon.Addon()
-       *  - self.Addon = xbmcaddon.Addon('script.foo.bar')
-       */
       Addon(const char* id = NULL);
-
       virtual ~Addon();
 
-      /**
-       * getLocalizedString(id) -- Returns an addon's localized 'unicode string'.
-       * 
-       * id        : integer - id# for string you want to localize.
-       * 
-       * example:
-       *   - locstr = self.Addon.getLocalizedString(32000)
-       */
+#ifdef DOXYGEN_SHOULD_USE_THIS
```

Comment:
I understand why it's happening. As was said in the other PR this is because you're trying to create python documentation from a c++ header.

Right now the entire file is in a weird dual mode where all code is ifdeffed out from doxygen. Is there even the point of having the docs in the same file now when code isn't used or attached to anything?